src/main/resources/templates/courseOffering/courseOffering.html

```diff
@@ -20,20 +20,22 @@ <h4>CSV must be in the following format:</h4>
 CMPSC 56,W20,Conrad,conrad@ucsb.edu
 CMPSC 48,F19,Richert Wang,richert@ucsb.edu
         </pre>
-        <br/>
+        <br />
         <form action="/courseOffering/upload" method="post" enctype="multipart/form-data">
           <input type="hidden" th:name="${_csrf.parameterName}" th:value="${_csrf.token}">
           <input type="file" name="csv" id="csv" accept="text/csv">
           <input type="submit" class="btn btn-primary" value="Submit">
         </form>
-        <br/>
+        <br />
         <table class="table table-striped table-responsive-md">
           <thead>
             <tr>
               <th>CourseId</th>
               <th>Quarter</th>
               <th>Instructor Name</th>
               <th>Instructor Email</th>
+              <th>Id</th>
```

Comment:
I don't think we need to display the id of each course, since that is database-specific and users shouldn't necessarily need to know about it. Unless there was a particular need you found to add this column?

Reply:
change made in branch